bpf/bpf_host.c

```diff
@@ -1125,6 +1122,10 @@ int to_host(struct __ctx_buff *ctx)
 	ret = CTX_ACT_OK;
 #endif /* ENABLE_HOST_FIREWALL */
 
+	if (!traced && !IS_ERR(ret))
```

Comment:
Since we are already checking for `IS_ERR(ret)` a few lines below, I think it would be better if we have this as an `else` branch on that check.

How about something like the following:

```
out:
     if (IS_ERR(ret)) {
            ...
     } else if (!traced) {
            ...
     }

Reply:
Thank you very much, that's helpful. I will try to fix that and commit again soon.